.github/workflows/cbbr_build.yml

```diff
@@ -74,9 +74,9 @@ jobs:
       - name: Export CBBR Output to DO (edm-publishing)
         working-directory: db-cbbr/cbbr_build
         run: ./04_export.sh
-        if: inputs.do_export == true
+        if: inputs.will_export == true
```

Comment:
very minor, but don't need `== true`!

Reply:
Also I didn't realize you didn't need these in `${{ inputs.foo }}` format

Reply:
yea I think I've only tried dropping the braces in these if clauses, pretty sure they're needed in more bash-ish places like `run: ...`